benchmarks/bench_fused_dit_layernorm.py

```
@@ -0,0 +1,195 @@
+def bench_mode(mode_name, eager_fn, fused_fn, shapes, device):
+    print(f"\n{'=' * 80}")
+    print(f"Mode: {mode_name}")
+    print(f"{'=' * 80}")
+    print(f"{'Shape':<40} {'Eager (ms)':>12} {'Fused (ms)':>12} {'Speedup':>10}")
+    print("-" * 80)
+
+    for batch_size, seq_len, desc in shapes:
+        d = make_inputs(batch_size, seq_len, device)
+
+        eager_times = bench_gpu_time(
+            lambda: eager_fn(d), enable_cupti=True, dry_run_iters=10, repeat_iters=100
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `d`

(B023)